api/v1/kogitobuild_types.go

```diff
@@ -68,70 +71,83 @@ type KogitoBuildSpec struct {
 	WebHooks []WebHookSecret `json:"webHooks,omitempty"`
 
 	// Native indicates if the Kogito Service built should be compiled to run on native mode when Runtime is Quarkus (Source to Image build only).
+	//
 	// For more information, see https://www.graalvm.org/docs/reference-manual/aot-compilation/.
 	// +optional
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Native Build"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Native Build"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	Native bool `json:"native,omitempty"`
 
 	// Resources Requirements for builder pods.
 	// +optional
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:resourceRequirements"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:resourceRequirements"
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 
 	// Maven Mirror URL to be used during source-to-image builds (Local and Remote) to considerably increase build speed.
 	// +optional
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Maven Mirror URL"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:label"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Maven Mirror URL"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
 
 	// Image used to build the Kogito Service from source (Local and Remote).
-	// The operator will use the one provided by the Kogito Team based on the "Runtime" field.
+	//
+	// If not defined the operator will use image provided by the Kogito Team based on the "Runtime" field.
```

Comment:
RHPAM Kogito Team ?

Reply:
maybe just, "will use the default builder image"